reactive/etcd.py

```diff
@@ -180,10 +180,12 @@ def configure_etcd():
                           cluster_data, owner='root', group='root')
     else:
         # render systemd
-        templating.render('systemd', 'files/systemd/etcd.service',
+        templating.render('systemd', '/etc/systemd/system/etcd.service',
                           cluster_data, owner='root', group='root')  # noqa
```

Comment:
Ah thats true. We only need to install this file if we've done this via resources. This did cause me some headache and I think I missed a patch I had made remotely to fix this. Good catch